README.md

```diff
@@ -37,45 +39,62 @@ NOTE: Currently, to access the API, you must be connected to the McGill network.
 ## Local installation
 
 ### Set the environment variables
-To run the API, at least two environment variables must be set, `USER` and `PASSWORD`. An optional third environment variable `DOG_ROOT` may be set to use a different IP address for the graph database.
+Create a `.env` file to house the environment variables used by the app. To run the API, at least two environment variables must be set, `USERNAME` and `PASSWORD`.  
+The contents of a minimal `.env` file:
+```bash
+USERNAME=someuser
+PASSWORD=somepassword
+```
 
-To set environment variables in macOS and Linux distributions:
+An optional third environment variable `DOG_ROOT` may be set in `.env` to use a different IP address for the graph database.
 
+To export all the variables in your `.env` file in one step, run the following:
 ```bash
-$ export KEY=value
-
-# For example
-$ export USER=someuser
+export $(cat .env | xargs)
 ```
 
-To set environment variables in Windows from CMD:
+The below instructions for Docker and Python assume that you have at least set `USERNAME` and `PASSWORD` in your current environment.
 
-```bash
-$ set KEY=value
+### Docker
+First, [install docker](https://docs.docker.com/get-docker/).
+
+ You can then run a Docker container for the API in three ways:
+#### Option 1: Use the `docker-compose.yaml` file
+
+First, [install docker-compose](https://docs.docker.com/compose/install/).
 
-# For example
-$ set USER=someuser
+If needed, update your .env file with optional environment variables for the docker-compose configuration:
+- `API_TAG`: Tag for API Docker image (default: `latest`)
+- `DOG_ROOT`: container name or IP address for the graph database (default: `graph`)
+- `STARDOG_TAG`: Tag for Stardog Docker image (default: `7.7.3-java11-preview`)
+- `STARDOG_ROOT`: Path to directory on host machine containing a Stardog license file (default: `~/stardog-home`)
+
+Note: To avoid conflicts related to [Docker's environment variable precedence](https://docs.docker.com/compose/environment-variables/envvars-precedence/), ensure that any variables defined in your `.env` file are not already set in your current shell environment with **different** values.
+
+Then spin up the containers using Docker Compose:
+```bash
+docker compose up -d
```

Comment:
you could add a note that you need to run this command in the same dir as the `docker-compose.yml` file, or you need to specify the path via `-f` (ask me how I know :) )